charts/merlin/templates/merlin-service-account.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if and .Values.serviceAccount.create }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+    name: {{ default "merlin" .Values.serviceAccount.name }}
+    namespace: {{ .Release.Namespace }}
+    {{- if and .Values.serviceAccount.annotations }}
```

Comment:
nit: this `and` here is not necessary right?

Reply:
yeah, i guess added by mistake.

Reply:
removed